scripts/apitypes-funcs/templates.go

```diff
@@ -130,6 +131,20 @@ func (obj *{{ .Type }}) SetConditions(conditions []metav1.Condition) {
 	obj.Status.Conditions = conditions
 }
 
+{{- end }}
+`
+
+	listFuncOutputFileNamme = "zz_generated_list_funcs.go"
+	listFuncTemplate        = `package {{ .PackageVersion }}
+
+// Code generated by scripts/apitypes-funcs/main.go; DO NOT EDIT.
+
+{{- range .Types }}
```

Comment:
```suggestion
{{ range .Types -}}
```
This way we should get the desired newline above the function comment.

Reply:
This was necessary after all. I've changed it in: e15f8000a771b6efc7519b9e052632cb1cbd1d91

Reply:
Hm, we still end up with the `// Code generated by scripts/apitypes-funcs/main.go; DO NOT EDIT.` comment stuck to a first `GetList` function. 🤔

Reply:
😬 I didn't catch that. 2679cbf6ba8ca9153f2620b9e9017dc5f3ae3f78 should hopefully be it.